python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -102,10 +107,16 @@ def setDefaults(self):
                               "base_PixelFlags",
                               "base_SkyCoord",
                               "base_PsfFlux",
+                              "base_GaussianCentroid",
+                              #"base_GaussianFlux",
```

Comment:
To run `base_GaussianFlux`, you'll need to run a shape algorithm (e.g. `base_SdssShape`), and set the shape slot to point to it (`...config.measurement.slots.shape = 'base_SdssShape'`).